akka-actor/src/main/scala/akka/actor/ClassicActorSystemProvider.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor
+
+import akka.annotation.DoNotInherit
+import akka.annotation.InternalApi
+
+/**
+ * Glue API introduced to allow minimal user effort integration between classic and typed for example for streams.
+ *
+ * Not for user extension.
+ */
+@DoNotInherit
+trait ClassicActorSystemProvider {
```

Comment:
maybe no need to prefix it with `Classic` since all things in this package is classic?

Reply:
I did it since it shows up in typed as well, and just having it named `ActorSystemProvider` there but not providing the `ActorySystem[_]` felt weird.

Reply:
ok